# .yaml config file
#
# The PID gains and controller settings must be saved in a yaml file that gets loaded
# to the param server via the roslaunch file (mouse_control.launch).

mouse:
  # Publish all joint states -----------------------------------
  # Creates the /joint_states topic necessary in ROS
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 30
  left_diff_drive_controller:
      type: effort_controllers/JointVelocityController
      joint: left_wheel_joint
      pid: {p: 1.0, i: 0.0, d: 0.0}
  right_diff_drive_controller:
      type: effort_controllers/JointVelocityController
      joint: right_wheel_joint
      pid: {p: 1.0, i: 0.0, d: 0.0}
